### PR TITLE
Add exercise tag-toggler component

### PR DESCRIPTION
**Pull request by dem4ron**
https://github.com/exercism/website/assets/66035744/cfcc2ac6-8748-49fc-9953-e97b85768151

### CODE REVIEW

File: app/javascript/components/track/exercise-community-solutions-list/ExerciseTagFilter.tsx [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+const toggledOffStyle =
+  'border-red text-red bg-[(var(--backgroundColorExerciseStatusTagLocked)]'
+function Tag({
+  tagName,
+  isActive,
+  onClick,
+}: {
+  tagName: string
+  isActive: boolean
+  onClick: (tag: string) => void
```
Comment (dem4ron) [inline app/javascript/components/track/exercise-community-solutions-list/ExerciseTagFilter.tsx]:
```suggestion
  onClick: (tagName: string) => void
```

**Comment by iHiD** [comment]
OK - this is great. 

It's in the wrong place though. This should live on a dedicated page in the maintainers UI.

It's missing the onclick callbacks back to the API from what I can see. (I know it's a draft, but thought I'd mention that here)

**Comment by iHiD** [comment]
@dem4ron This will stay as a maintainers function, not part of the ml_trainer thing.

**Comment by iHiD** [comment]
@dem4ron Can this be closed now?

**Pull request closed without merging**